rosdep/base.yaml

```diff
@@ -6982,6 +6982,9 @@ python-dev:
   debian: [python-dev]
   fedora: [python2-devel]
   nixos: [pythonPackages.python]
+  rhel:
+    '7': [python2-devel]
```

Comment:
@cottsay isn't this one just `python-devel`?

Reply:
RHEL has no dashboard to demonstrate this, but I can show it using command line tools on CentOS 7. The package is called `python-devel`, but it provides a virtual package for `python2-devel`:
```
$ rpm --qf '[%{PROVIDES}\n]' -q python-devel
pkgconfig(python)
pkgconfig(python-2.7)
pkgconfig(python2)
python-devel
python-devel(x86-64)
python2-devel
python2-devel(x86-64)
```

I'm inclined to use the more specific virtual package here because modern Fedora is starting to provide the unversioned form with Python 3 packages.